components/NewsItem.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react'
 import Upvote from './Upvote'
+import Points from './Points'
 
 class NewsItem extends React.Component {
   constructor (props, context) {
     super(props, context)
     this.state = {upvoted: this.props.upvoted}
+    this.state = {count: this.props.count}
```

Comment:
Here you're overwriting `this.state`, rather than adding an extra attribute to it. Should be more like:

```js
this.state = {
  count: this.props.count,
  upvoted: this.props.upvoted
}
```